### ISSUE TITLE
Bug: Previous example from RLZoo now has a bug

### ISSUE THREAD

**Issue by tyler-ingebrand** [JuliaReinforcementLearning/ReinforcementLearning.jl#643]
I am attempting to use https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/master/src/ReinforcementLearningExperiments/deps/experiments/experiments/DQN/JuliaRL_BasicDQN_MountainCar.jl as an example and starting point for some code. After trying to run it, I get an error:
```
ERROR: LoadError: MethodError: no method matching QBasedPolicy(::BasicDQNLearner{NeuralNetworkApproximator{Chain{Tuple{Dense{typeof(relu), CUDA.CuArray{Float32, 2, CUDA.Mem.DeviceBuffer}, CUDA.CuArray{Float32, 1, CUDA.Mem.DeviceBuffer}}, Dense{typeof(relu), CUDA.CuArray{Float32, 2, CUDA.Mem.DeviceBuffer}, CUDA.CuArray{Float32, 1, CUDA.Mem.DeviceBuffer}}, Dense{typeof(identity), CUDA.CuArray{Float32, 2, CUDA.Mem.DeviceBuffer}, CUDA.CuArray{Float32, 1, CUDA.Mem.DeviceBuffer}}}}, ADAM}, typeof(huber_loss), StableRNGs.LehmerRNG}, ::EpsilonGreedyExplorer{:exp, false, StableRNGs.LehmerRNG})

Closest candidates are:
  QBasedPolicy(::Q, ::E) where {Q<:AbstractLearner, E<:AbstractExplorer} at ~/.julia/packages/ReinforcementLearningCore/XT5LC/src/policies/q_based_policies/q_based_policy.jl:16
```
I know this example should work because I have used it in the past. It is claiming a BasicDQNLearner is not of type AbstractLearner. However, it is defined to be here: https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/f4cf555f50e0333254ab099f5a643a587ec532d7/src/ReinforcementLearningZoo/src/algorithms/dqns/basic_dqn.jl

I suspect the issue is in the includes/usings. Probably, the AbstractLearner code is included twice (in two modules), and the modular the function call checks against is not the same as the module that can produce the BasicDQNLearner. I have seen issues in the past where the same code is in two modules, and the compiler considers included code to be two different modules even though they are the same code. I could be wrong though.

Versions:
```
  [158674fc] ReinforcementLearning v0.10.0
  [860ef19b] StableRNGs v1.0.0
  [587475ba] Flux v0.12.4
  [91a5bcdd] Plots v1.29.0
  Julia 1.7.2
```

Let me know if you need any other details from me.

**Comment by tyler-ingebrand** [comment]
Getting similar type errors for https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/master/src/ReinforcementLearningExperiments/deps/experiments/experiments/Policy%20Gradient/JuliaRL_DDPG_Pendulum.jl

**Comment by tyler-ingebrand** [comment]
It seems to have been some sort of version inconsistency on my part. I deleted .julia and reinstalled everything. My packages became:
```
  [158674fc] ReinforcementLearning v0.10.0
  [8197267c] IntervalSets v0.5.4
  [91a5bcdd] Plots v1.29.0
  [587475ba] Flux v0.12.10
  [860ef19b] StableRNGs v1.0.0
```
and it is working now. I am not sure how that happened. I did use 
`Pkg.add("ReinforcementLearning");   using ReinforcementLearning`
for each package to automatically install them if they are not installed already. Perhaps that is a bad idea.
